locale/sv/LC_MESSAGES/django.po

```diff
@@ -493,15 +502,42 @@ msgstr "Faktura"
 msgid "Invoices"
 msgstr "Fakturor"
 
-msgid "Lease"
-msgstr "Arrendering"
-
 msgid "Lease identifier"
 msgstr "Arrendekod"
 
+msgid "Invoice number"
+msgstr "Fakturanummer "
+
 msgid "Due date"
 msgstr "Förfallodag"
 
+msgid "Service unit"
+msgstr ""
```

Comment:
Should we get translations to these?

Reply:
This was generated automatically so I'm not sure

Reply:
Now that I check there are quite a few translations missing for Swedish such as "Service unit". Would probably be good to get translations. I don't know if I feel comfortable enough with my Swedish skills to try to add these. My best guess for service unit is "servicepaket" 😄